tests/run_tests.py

```diff
@@ -0,0 +1,46 @@
+import os
+from rockstarpy.rockstar import convert_code
+import difflib
+from pprint import pprint
+try:
+    from StringIO import StringIO # python 2
+except:
+    from io import StringIO  # python 3
+
+
+def check_files_identical(expected, actual):
+    diff = difflib.unified_diff(
+                expected,
+                actual,
+                fromfile='expected',
+                tofile='actual',
+    )
+    diff = list(diff)
+    if len(diff):
+        for d in diff:
+            print(diff)
+        assert False, "There are differences"
+
+def main():
+    files = os.listdir('.')
+    rock_files = filter(lambda f: '.rock' == f[-5:], files)
+    py_files = set(filter(lambda f: '.py' == f[-3:], files))
+    for rock_file in rock_files:
+        file_name = rock_file[:-5]
```

Comment:
you are assuming that all rockstar files uses ".rock" extension, but on the help for the argument "input" there are listed three different extensions `(rock|rockstar|lyrics)`.
I think that `rock_file.rstrip('.', 1)[0]` is better